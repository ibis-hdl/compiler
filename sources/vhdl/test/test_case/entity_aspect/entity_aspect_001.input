entity entity_name ( architecture_identifier )
