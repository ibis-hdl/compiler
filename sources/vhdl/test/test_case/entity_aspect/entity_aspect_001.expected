(entity_aspect<v>
  (name<v>
    (identifier
      COUNTER
    )
  )
)
